chia/rpc/full_node_rpc_api.py

```diff
@@ -140,14 +172,29 @@ async def get_blockchain_state(self, _request: Dict):
 
         if self.service.mempool_manager is not None:
             mempool_size = len(self.service.mempool_manager.mempool.spends)
+            mempool_cost = self.service.mempool_manager.mempool.total_mempool_cost
+            mempool_min_fee_5m = self.service.mempool_manager.mempool.get_min_fee_rate(5000000)
         else:
             mempool_size = 0
+            mempool_cost = 0
+            mempool_min_fee_5m = 0
         if self.service.server is not None:
             is_connected = len(self.service.server.get_full_node_connections()) > 0
         else:
             is_connected = False
         synced = await self.service.synced() and is_connected
 
+        compact_blocks = 0
+        uncompact_blocks = 0
+        with log_exceptions(self.service.log):
+            compact_blocks = await self.service.block_store.count_compactified_blocks()
```

Comment:
We have to ensure this is fast, because `get_blockchain_state` might get called very often by people. If it's not fast, I suggest we move it somewhere else where it can get called more rarely.

Reply:
I'm fine moving some of this stuff to a different RPC call, was just trying to fit in to existing calls if the data seemed to make sense.. I did time this at one point, but forget how quick it was - was a few ms IIRC, but will redo that and document in this PR

Reply:
Moved this + hint counts to a new RPC call.

The slowest I observed all three calls to be was about 12ms combined. most of the time, they were closer to the 1-2ms range - This is on a relatively quick machine with an SSD, so could be a bit slower on an HDD or something slower.

Either way, being it its own endpoint now probably alleviates concerns about this running when people may not need this data.